app/src/layouts/kanban/index.ts

```diff
@@ -353,7 +353,7 @@ export default defineLayout<LayoutOptions, LayoutQuery>({
 			function createViewOption<T>(key: keyof LayoutOptions, defaultValue: any) {
 				return computed<T>({
 					get() {
-						return layoutOptions.value?.[key] ?? defaultValue;
+						return layoutOptions.value?.[key] !== undefined ? layoutOptions.value?.[key] : defaultValue;
```

Comment:
Removing unnecessary optional chaining operator:
```suggestion
						return layoutOptions.value?.[key] !== undefined ? layoutOptions.value[key] : defaultValue;
```

Reply:
Thank you! Updated for `cards` layout too.

Reply:
Perfect 🚀